opentok/opentok-meet
# [OPENTOK-44303] Add screen to canvas functionality

#### What is this PR doing?
Adding screen to canvas functionality.

#### How should this be manually tested?
Run meet locally and:
Test 1
- Click Screen and check ClientQoS the videoSource for its publisher is 'screen'.
- Stop ScreenSharing and click now on ScreenToCanvas and check ClientQoS, videoSource is now 'custom' for the publisher. Check videoWidth is 1280 and videoHeight is 720.

Test 2
- Check both options are available from the beginning: Screen and ScreenToCanvas
- Click in one of them. Check now only the selected option is now visible.
- Click again in the same button (stop sharing). Check the 2 options are now back.

Test 3
- Click in ScreenToCanvas, and finish the sharing by clicking in the browser pop-up: Stop sharing.
- Check publisher successfully closed.

#### What are the relevant tickets?
OPENTOK-44303
